yarn.lock

```diff
@@ -11963,9 +11963,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#0387db93d7fad4ac530afe0a5a4e5cd91cde8c99":
-  version "3.136.6"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#0387db93d7fad4ac530afe0a5a4e5cd91cde8c99"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#76083e33f175fb00392e31f1f5f90654d05f1fd2":
+  version "3.137.7"
```

Comment:
[This PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/336/files) changed the schema version from `3.136.6` to `3.137.6`, which is why the jump looks weird.